website/static/js/pages/project-settings-page.js

```diff
@@ -147,6 +151,34 @@ $(document).ready(function() {
                 }
             }
         });
+        };
```

Comment:
All methods that do http work [should](http://cosdev.readthedocs.org/en/latest/style_guides/javascript.html#promises) return a promise or deferred. That way, things that call submit could add additional actions if desired. (using `jqXHR.done()` or `jqXHR.fail()`; see [docs](https://api.jquery.com/jquery.ajax/#jqXHR))

This is automatically returned when by `$.ajax`, so it's an easy change. You might write:

```
request = $.ajax(...);
return request;
```